people-and-technology/physical.md

```diff
@@ -0,0 +1,93 @@
+- `Tab` to move from one focussable item to the next
+- `Enter` to activate alink or button
+- `Space` to select or deselect a checkbox
+- `Esc` to cancel a prompt
+
+### Guidance
+
+- Ensure interactive controls are focussable using the `Tab` key.
+- Keep the tab order logical. Ensure focus doesn't skip unpredictably around the
+  page while tapping the `Tab` key.
+- It should be immediately obvious which element is currently in focus. Ensure
+  the focus state of each interactive element is clear.
+- If there are a lot of navigation controls at the top of the page, provide a
+  [skip link](skip-link) that takes the user to the main content.
+
+_Note:_ if you are using Firefox on Mac you need to make some changes to [make tab behaviour work as expected](https://stackoverflow.com/questions/11704828/how-to-allow-keyboard-focus-of-links-in-firefox.).
+
+[skip-link](https://design-system.service.gov.uk/components/skip-link/)
+
+### Further resources
+
+- [Source's developer guidelines on keyboard navigation](https://github.com/guardian/source/blob/main/docs/06-accessibility.md#keyboard-navigation)
+- [How to implement websites that are ready for keyboard only usage](https://www.accessibility-developer-guide.com/knowledge/keyboard-only/how-to-implement/)
+
+## Switch controls
+
+People with limited or no upper body movement may use switch controls, as well
+as people with cognitive and learning difficulties.
+
+There are several types of switches that accommodate different needs:
+
+- Sip-and-puff switches are "clicked" by sipping and puffing into a straw
+- Button switches can be activated using the hand, foot or head. It is
+  possible to turn your keyboard into a button switch by adjusting your
+  computer's accessibility settings.
+- Camera switches can be activated when tilting the head into a camera
+- Eye tracking controls can measure where a user is looking, or track the motion
+  of an eye relative to the head.
+
+### Guidance
+
+Much of the guidance that is true for keyboard controls is also true for switch
+controls. There are also some time-based considerations:
+
+- Typing using switches is much slower than using a keyboard, with users
+  typically entering between 1 and 2 words per minute. Consider implementing
+  features that reduce the need for typing, such as autocompletion or word
+  prediction.
+- Avoid imposing time limits on form completion or confirmation controls.
+- Scrolling can be slow as operating systems tend to bury the scroll controls.
+  Product developers should try to make important navigation and information
+  appear above the fold, to reduce the impact of scroll fatigue.
+- Switch users may spend a long time completing text fields on forms. If a
+  browser crashes, or the user accidentally closes a tab or hits the Back
+  button, it can be very frustrating to lose all that typing. A feature that
+  automatically saves work in progress can save a lot of time.
```

Comment:
I'd suggest adding something about making it easy to go back and correct mistakes on forms too- people can be more likely to make typos when using switches.

Reply:
Good thought, I'll add it!